### ISSUE TITLE
Analog of `cargo dev new_lint`

### ISSUE THREAD

**Issue by smoelius** [trailofbits/dylint#31]
In #28, @danielhenrymantilla wrote:
> (I'd even suggest making a `cargo dylint new my_lint` subcommand which would basically do a `(git clone …/dylint-template my_lint && cd my_lint && bash script_name.sh my_lint)` + print on stdout the workspace metadata lines to load it by path, with a commented out repo line).

This is a really good idea, IMHO.

Clippy's `cargo dev new_lint` is discussed here: https://github.com/rust-lang/rust-clippy/blob/master/doc/adding_lints.md#getting-started

**Comment by smoelius** [comment]
I'm leaning toward `cargo dylint --init <name>`, and then the clone is to the current directory.

**Comment by danielhenrymantilla** [comment]
Yeah, I don't mind it either way 👍 I still suggest that part of `cargo vendor`-like behavior, where the extra necessary setup (`.config` file for `cargo vendor` / Cargo.toml `metadata` in our case for lint users) is printed to stdout while the meat of the command lays out the directory contents.
